### PR TITLE
update homepage, footer, and README to be more specific to nwalsdev

### PR DESCRIPTION
**Pull request by bjkeeton**
I updated the README with basic project functionality as I saw them.

Additionally, I removed the Pinax default homepage text and updated it to be specific to us and what the group is. It's still not pretty at this point, too, but it's not the default template anymore. I also fixed the footer to not be generic placeholder.

### CODE REVIEW

File: nwalsdev/templates/_footer.html [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {% load i18n %}
 
-{% trans "&copy; 2016 &lt;your company here&gt;" %}
+{% trans "&copy; 2016 Northwest Shoals Software Developers" %}
```
Comment (jakesen) [inline nwalsdev/templates/_footer.html]: Should be `Northwest Alabama Software Developers`...

**Comment by jakesen** [comment]
This is great! So glad you decided to help out on this...especially since I haven't had much time to work on it.

Were you able to get a local environment working? If so, do you have a screen shot of the updated home page that you could post here?

**Comment by bjkeeton** [comment]
I'm working on getting it set up. I have no idea why Django hates me so much. It doesn't like the default directory structure that apt installed Django and Virtualenv into. #eyeroll 

I hope I can get it working tonight.

**Comment by bjkeeton** [comment]
screenshot from 2016-08-07 17-47-07

This is the super basic template. I think my question at this point is how do I remove the "example.com" from the header. Nothing I do turns up where it's being imported from, even though I could find the site_title and change it. I am obviously missing something, but there's no reason this should be called "example.com"

File: nwalsdev/templates/account/login.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {% extends "site_base.html" %}
 
-{% load account_tags %}
```
Comment (jakesen) [inline nwalsdev/templates/account/login.html:3]: Why is this being removed? If we don't need it, is there any reason to leave the empty code brackets?

**Comment by jakesen** [comment]
Looks like you got your local working! "example.com" is coming from Django's `Site` model. If you want to take a gander at what's in your local database (or change it), you can create a super user and log into the Django admin:

```
python manage.py createsuperuser
```

Then go to `localhost:8000/admin` in your browser.

**Comment by jakesen** [comment]
Oh, looks like `mydatabase` and `requests` should not have been committed.

File: nwalsdev/templates/account/login.html (hunk 2 above)
Comment (bjkeeton) [inline nwalsdev/templates/account/login.html:3]: That's a typo, actually, from when I was experimenting to remove the example.com stuff (couldn't find your answer in the documentation).

**Comment by bjkeeton** [comment]
nwalsdev2

And I think I got the Tech256 links working, as well as adding the mdatabase and requests to my .gitignore. :)

File: .gitignore [hunk 3]
```diff
@@ -3,3 +3,6 @@ node_modules/
 _build/
 dev.db
 site_media/
+env
+mydatabase
+requests
```
Comment (jakesen) [inline .gitignore]: No newline

**Comment by jakesen** [comment]
It looks like `mydatabase` and `requests` still need to be removed from the branch since they have already been added.

File: .gitignore (hunk 3 above)
Comment (bjkeeton) [inline .gitignore]: I'm not sure I follow this one, sorry.

Comment (jakesen) [inline .gitignore]: Although it usually doesn't cause any problems, source code files should always end with a newline: https://robots.thoughtbot.com/no-newline-at-end-of-file

Comment (bjkeeton) [inline .gitignore]: Makes sense. I fixed the Sublime3 setting so it'll be automatic from now on. :)

**Comment by jakesen** [comment]
Looks great! I'm gonna merge it! 👍

**Pull request merged**